### DISCUSSION TITLE
can't scrolling

### DISCUSSION THREAD

**Discussion by chenjijun** [flet-dev/flet#3418] [General]
when i creat a first column container  and creat a second column container in the first container ,then  creat a third row container. when i use scroll control in the third container it doesn't work. only can use scorll in first container

**Comment by ndonkoHenri** [comment]
Please share some runnable code for better understanding of your concern.

**Comment by chenjijun** [comment]


**Reply by ndonkoHenri** [reply]
The code is not runnable. Can you share some directly-runnable code I can use to test?
